content/200-concepts/100-components/02-prisma-client/060-full-text-search.mdx

```diff
@@ -18,24 +18,24 @@ The full-text Search API is still in **Preview**. You can enable `fullTextSearch
 
 1: Update the [`previewFeatures`](../preview-features) block in your schema:
 
-    ```prisma
-    generator client {
-      provider        = "prisma-client-js"
-      previewFeatures = ["fullTextSearch"]
-    }
-    ```
+```prisma
+generator client {
+  provider        = "prisma-client-js"
+  previewFeatures = ["fullTextSearch"]
+}
+```
 
 2: Generate the Prisma Client:
 
-    ```terminal copy
-    npx prisma generate
-    ```
+```terminal copy
+  npx prisma generate
+```
```

Comment:
I think a better solution here would be to keep the indentation of the code blocks, and replace `1:` and  `2:` with `1.` and `2.` to make it a numbered list. Then this would look fine.